pcdsdevices/smarpod.py

```diff
@@ -151,3 +213,61 @@ class SmarPod(Device):
     cmd_calibrate = Cpt(EpicsSignal, ':CMD:CALIBRATE', kind='normal')
     cmd_read_ver = Cpt(EpicsSignal, ':CMD:READ_VER', kind='normal')
     ver_sys = Cpt(EpicsSignalRO, ':VER:SYS', kind='omitted')
+    # coordinate offset
+    csy_set = Cpt(EpicsSignal, ':CSY:SET', kind='normal', doc="set csy offset")
+    csy_zero = Cpt(EpicsSignal, ':CMD:SPZ', kind='normal', doc="set csy to zero")
+
+    def multi_axis_move(
+        self,
+        x_sp: float | None = None,
+        y_sp: float | None = None,
+        z_sp: float | None = None,
+        rx_sp: float | None = None,
+        ry_sp: float | None = None,
+        rz_sp: float | None = None,
+        wait: bool = True,
+        timeout: float = 30.0,
+    ):
+        x_sp = self.x.readback.get() if x_sp is None else x_sp
+        y_sp = self.y.readback.get() if y_sp is None else y_sp
+        z_sp = self.z.readback.get() if z_sp is None else z_sp
+        rx_sp = self.rx.readback.get() if rx_sp is None else rx_sp
+        ry_sp = self.ry.readback.get() if ry_sp is None else ry_sp
+        rz_sp = self.rz.readback.get() if rz_sp is None else rz_sp
+
+        self.x.defer_motion = True
+        self.y.defer_motion = True
+        self.z.defer_motion = True
+        self.rx.defer_motion = True
+        self.ry.defer_motion = True
+        self.rz.defer_motion = True
+
+        self.x.setpoint.put(x_sp)
+        self.y.setpoint.put(y_sp)
+        self.z.setpoint.put(z_sp)
+        self.rx.setpoint.put(rx_sp)
+        self.ry.setpoint.put(ry_sp)
+        self.rz.setpoint.put(rz_sp)
+
+        x_status = self.x.move(x_sp, wait=False, timeout=timeout)
+        y_status = self.y.move(y_sp, wait=False, timeout=timeout)
+        z_status = self.z.move(z_sp, wait=False, timeout=timeout)
+        rx_status = self.rx.move(rx_sp, wait=False, timeout=timeout)
+        ry_status = self.ry.move(ry_sp, wait=False, timeout=timeout)
+        rz_status = self.rz.move(rz_sp, wait=False, timeout=timeout)
```

Comment:
Question: what's the purpose of putting to the setpoints manually and also calling move?